package.json

```diff
@@ -575,10 +575,15 @@
                         "description": "%azFunc.pickProcessTimeoutDescription%",
                         "default": 60
                     },
-                    "azureFunctions.showFuncInstallation": {
-                        "type": "boolean",
-                        "description": "%azFunc.showFuncInstallationDescription%",
-                        "default": true
+                    "azureFunctions.backupTemplateRuntime-~1": {
+                        "type": "string",
+                        "description": "%azFunc.backupTemplateRuntime-~1%",
+                        "default": "1.0.12.1"
+                    },
+                    "azureFunctions.backupTemplateRuntime-beta": {
```

Comment:
I don't like making a specific setting for each runtime because then we're forced to add a new setting anytime the runtime version changes. (For example they will stop using 'beta' at some point and switch to '~2').

I think you can just do one generic setting that applies to all runtimes. People can't use multiple runtimes at the same time anyways. I suggest simply 'templateVersion'

Reply:
The current behavior of the function makes us download both runtimes though.  Which release should I download for the runtime that is not specified?  Just default to the latest?

Reply:
"The current behavior of the function" lol why does that matter? You have the ability to change EVERYTHING

I see a few options:
1. As you mentioned: default to the latest for the runtime that's not specified (which I'd be fine with).
1. Use the version specified for BOTH runtimes (I could see an advanced user wanting to use the v1 JavaScript templates even on 'beta'. There's not that much difference in JavaScript and more exist for v1)